### PR TITLE
New Tests for selectors

### PR DESCRIPTION
**Pull request by jmjames37**
New Tests for Phase 1

### CODE REVIEW

**Comment by jmjames37** [comment]
Double Check

**Review by GeorgeLuo — APPROVED** [review]
looks good to me

File: src/test/java/dev/amp/validator/SrcsetSourceDefTest.java [hunk 1]
```diff
@@ -0,0 +1,16 @@
+package dev.amp.validator;
+
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class SrcsetSourceDefTest {
+
+    @Test
+
```
Comment (GeorgeLuo) [inline src/test/java/dev/amp/validator/SrcsetSourceDefTest.java:9]: remove this space

File: src/test/java/dev/amp/validator/utils/ParseSrcSetUtilsTest.java [hunk 2]
```diff
@@ -31,7 +35,10 @@
 
 public class ParseSrcSetUtilsTest {
 
-  @Test
-  public void testParseSrcset() {
-  }
+    @Test
+    public void testParseSrcset() {
+        SrcsetParsingResult a = parseSrcset("google.com/favicon.ico");
+
```
Comment (GeorgeLuo) [inline src/test/java/dev/amp/validator/utils/ParseSrcSetUtilsTest.java:41]: remove these spaces

**Pull request closed without merging**